.travis.yml

```diff
@@ -8,21 +9,21 @@ install:
 - (cd integration-tests && bundle install)
 
 script:
+# Build & run tests
```

Comment:
This was after what is now line 16 on purpose. We didn't want the integration tests speaking with Ringcaptcha.